bin/local/nuke-local.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+### failsafe logic to exit in case we are not running in our local minikube context
+KUBE_CONTEXT=$(kubectl config current-context)
```

Comment:
Suggestion: as this could be helpful in basically all other scripts as well, would it be a thing to factor it out into a file other scripts can source, with an API like this:

```
source ./require-env.sh
require_env "local"
# or
require_env "staging" "production"
```

Reply:
yeah I'm super torn apart about stuff like this. Personally I deeply love it and even used something like this at first, but keeping our codebase complexity in mind I like to think that it's easier maintainable and understandable if we keep these self-contained :thinking: I guess there's a possibility for an ADR discussion hidden here somewhere